docs/examples/getting started.ipynb

```diff
@@ -58,7 +58,7 @@
       " |  \n",
       " |  define_graph(self)\n",
       " |      # defined by the user to construct their graph of operations.\n",
-      " |      # this returns a list of output TensorReferences that we can\n",
+      " |      # this returns a list of output EdgeReference that we can\n",
```

Comment:
Could you regenerate this? I think this docstring looks different now.

Reply:
Done